aws/v9.3.6/README.md

```diff
@@ -0,0 +1,27 @@
+# :zap: Giant Swarm Release v9.3.6 for AWS :zap:
+
+**If you are upgrading from 9.3.5, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release updates managed apps to the latest releases.
+
+Below, you can find more details on components that were changed with this release.
+
+### cluster-operator [0.23.13](https://github.com/giantswarm/cluster-operator/releases/tag/v0.23.13)
+
+- Enable NGINX App managed NodePort Service on KVM.
+
+### kube-state-metrics v1.9.7 ([Giant Swarm app v1.1.1](https://github.com/giantswarm/kube-state-metrics-app/blob/master/CHANGELOG.md#111---2020-07-22))
+
+- Updated kube-state-metrics version from 1.9.5 to 1.9.7. Check the [upstream changelog](https://github.com/kubernetes/kube-state-metrics/blob/master/CHANGELOG.md#v197--2020-05-24) for details on all changes.
+
+### metrics-server v0.3.6 ([Giant Swarm app v1.1.1](https://github.com/giantswarm/metrics-server-app/blob/master/CHANGELOG.md#111---2020-07-23))
+
+- Updated metrics-server version from 0.3.3 to 0.3.6. Check the [upstream changelog](https://github.com/kubernetes-sigs/metrics-server/releases) for details on all changes.
+
+### nginx-ingress-controller v0.34.1 ([Giant Swarm app v1.8.1](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#181---2020-07-28))
+
+- Drop support for deprecated configuration properties.
```

Comment:
@sslavic might this break anything for customers? I only noticed the dropping of support for configs. How do we know if customers are still using these deprecated configs in their workflow?

Reply:
thank you!